src/modules/profile/ProfileRoutes.tsx

```diff
@@ -39,6 +51,12 @@ export function ProfileRoutes() {
         exact={true}
         component={LoadableProfileContainer}
       />
+
+      <PrivateRoute
```

Comment:
Something wrong with this route. When route is '/profile/edit' component for '/profile' is visible too.

Reply:
I think, I resolve this with wrap these routes in <Switch>. However it was strange